src/datajudge/constraints/base.py

```diff
@@ -123,6 +127,7 @@ def __init__(
         output_processors: Optional[
             Union[OutputProcessor, List[OutputProcessor]]
         ] = output_processor_limit,
+        lru_cache_maxsize=None,
```

Comment:
I think using `lru_cache_maxsize` is a bit of a "mouthful". Would it work if it's `cache_size` instead? I'd say that the fact it's lru it's more of an implementation detail than something relevant to the user. You can tuck the `lru` and `max` parts away in the documentation for the curious.

Reply:
Done with f14a54b